### ISSUE TITLE
Emphasis current hour

### ISSUE THREAD

**Issue by CirruZZ** [hotNipi/node-red-contrib-ui-statechart#4]
Using the shortcut 24H|L to highlight the current hour is a bit vague. I'w tried a bit different colors but its tricky to get a clear view. Works best with darker colors.

![2](https://user-images.githubusercontent.com/480883/204484786-4d391020-ab16-4709-af72-2e2b42c29467.png)

Suggestion is to emphasis it with bold text and som outline/border. 

![1](https://user-images.githubusercontent.com/480883/204485799-25f732a6-681f-4676-a365-80aaaf96f105.png)

**Comment by hotNipi** [comment]
And with 0.2.0-beta.4 a little bit different emphasis

**Comment by CirruZZ** [comment]
> And with 0.2.0-beta.4 a little bit different emphasis

Not my type, it works but makes it to "fancy", if you ask me.

Could just be bold text, that's enough.

![3](https://user-images.githubusercontent.com/480883/204522015-9b917b97-27ba-473e-b713-cef020d7a77f.png)


> And with 0.2.0-beta.4 a little bit different emphasis

**Comment by hotNipi** [comment]
Well the benefit is that it does not require any color adjustment, it works for custom colors and does not depend on theme. Bold takes more space in area thus the overlapping texts may occur too easily. 

A bit of background ..
I have user (a good friend of mine) with exact requirements on widget layout and sizes. So it lives in production environment. I actually developed this for him, taking account every requirement he did pointed out. Then as it turned out useful enough I shared in public. Yes, I may say that do not update but hey ...
